addons/web/static/tests/control_panel/control_panel_model_extension_tests.js

```diff
@@ -369,5 +369,52 @@ odoo.define("web/static/tests/control_panel/control_panel_model_extension_tests.
 
         });
 
+        QUnit.test('search defaults on X2M fields', async function (assert) {
+            assert.expect(1);
+
+            const context = {
+                search_default_otom: [1, 2],
+                search_default_mtom: [1, 2]
+            };
+            const fields = this.fields;
+            fields.otom = { string: "O2M", type: "one2many", relation: 'partner' };
+            fields.mtom = { string: "M2M", type: "many2many", relation: 'partner' };
+            const arch = `
+                <search>
+                    <field name="otom"/>
+                    <field name="mtom"/>
+                </search>`;
+            const model = createModel({ arch, fields, context });
+            assert.deepEqual(sanitizeFilters(model), [
+                {
+                    "defaultAutocompleteValue": {
+                      "label": [1, 2],
+                      "operator": "ilike",
+                      "value": [1, 2]
```

Comment:
Not a request for changes but a tip to make your life easier next time: you could have directly compared the `defaultAutocompleteValue.value` since it's the only interesting part in the filter data for this test, and other tests here already assert the other info. I realize thought that it wasn't obvious at all since the control panel model's logic is quite intricate 😄

Reply:
lol I was going to do it but since all asserts in the file compare the whole result of `sanitizeFilters(model)`, I let the same comparaison. Good to know for the next time. Thanks!